biztalk/TOC.md

```diff
@@ -2609,6 +2609,7 @@
 ####### [Change the Console Colors for BTSTask](core/how-to-edit-the-console-colors-for-btstask.md)
 #### [Restart services, shut down BizTalk Server](core/how-to-start-stop-pause-resume-or-restart-biztalk-server-services.md)
 #### [Audit management operations](core/audit-management-operations.md)
+#### [Enable Receive Location fault tolerance](technical-guides/how-to-enable-receive-location-fault-tolerance)
```

Comment:
Pages under 'technical-guides' folder should be added under the Technical Guides section in the TOC.
If you want to add this page under in the Core docs in TOC, please move your page to 'core' folder

Reply:
Also, while including the page in TOC, make sure to have the '.md' file extension